sdk/go/common/workspace/workspace.go

```diff
@@ -105,7 +105,7 @@ func NewFrom(dir string) (W, error) {
 
 	err = w.readSettings()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "unable to read workspace settings")
```

Comment:
`fmt.Fprintf("unable to read workspace settings: %w", err)` I thought maybe is the more modern form (nit)

Reply:
Aside: I realize it's a low-value return, but at some point I'd love to remove our dependency on `github.com/pkg/errors` and move to the more modern built-in equivalents throughout our codebase. Maybe when we have a little downtime near the holidays we can look into it. Seems like there are tools that can help with this: https://xdg.me/rewriting-go-with-ast-transformation/